packages/atomic/create-atomic/scripts/packageTemplate.json

```diff
@@ -17,6 +17,6 @@
     "site:init": "netlify init",
     "site:link": "netlify link",
     "site:deploy": "npm run site:link && npm run build && netlify env:import .env && netlify deploy --prod --open",
-    "postinstall": "npm run setup-lambda && npm run setup-cleanup"
+    "postinstall": "npm run setup-lambda && npm run setup-cleanup && prettier --write . --loglevel warn"
```

Comment:
We'd have to do this for all files that contain markup , not just the one we writefile with. It would have to be on each plop method with a `transform` perhaps.

I'd rather do it on the project as a whole once it's installed. Then the user still has prettier installed & configured in his to format his project as needed, which is a plus. End result is the same too.

What do you think?

Reply:
Yeah sure then, than make sense